nginx.conf

```diff
@@ -88,14 +88,16 @@ http {
         rewrite ^/docs/([a-zA-Z][a-zA-Z])/arbitrum-faq/$                                $scheme://$http_host/docs/$1/arbitrum/arbitrum-faq/                         permanent;
         rewrite ^/docs/([a-zA-Z][a-zA-Z])/cookbook/migrating-a-subgraph/$               $scheme://$http_host/docs/$1/cookbook/upgrading-a-subgraph/                 permanent;
         rewrite ^/docs/([a-zA-Z][a-zA-Z])/cookbook/quick-start/$                        $scheme://$http_host/docs/$1/quick-start/                                   permanent;
-        rewrite ^/docs/en/substreams/(?!index\.).+$                                     https://substreams.streamingfast.io                                         permanent;
-        rewrite ^/docs/en/firehose/(?!index\.).+$                                       https://firehose.streamingfast.io                                           permanent;
```

Comment:
Just moved these to the bottom of the list because they're different from the other (English-only).